src/bigscreenplayer.js

```diff
@@ -484,13 +530,13 @@ function BigscreenPlayer() {
      * @function
      * @param {*} opts
      * @param {boolean} opts.userPause
-     * @param {boolean} opts.disableAutoResume
      */
     pause: (opts) => {
       DebugTool.apicall("pause")
 
-      pauseTrigger = opts && opts.userPause === false ? PauseTriggers.APP : PauseTriggers.USER
-      playerComponent.pause({ pauseTrigger, ...opts })
+      pauseTrigger = opts?.userPause || opts?.userPause == null ? PauseTriggers.USER : PauseTriggers.APP
```

Comment:
We want `pauseTrigger` to be `PauseTriggers.USER` when `opts.userPause` is `null` or `undefined`? Is this correct?

If so - previous comment about Double Equals also applies.